packages/live-share-media/src/VolumeLimiter.ts

```diff
@@ -3,59 +3,60 @@
+import { IMediaPlayer } from './IMediaPlayer';
 
 export enum LevelType { fixed, percentage }
-
 export class VolumeLimiter {
     private readonly _player: IMediaPlayer;
+    private readonly _rampDuration = new TimeInterval(500);
 
-    // Settings
-    private readonly _duration = new TimeInterval(2000);
-    private readonly _rampDownDuration = new TimeInterval(500);
-    private readonly _rampUpDuration = new TimeInterval(500);
-    private _level: number = 0.1
+    private _selectedVolume = 1.0;
+    private _limited = false;
+    private _level = 0.1
     private _levelType: LevelType = LevelType.fixed;
-
-    // Running state
-    private _running = false;
-    private _endTime = 0;
     private _startTime = 0;
     private _startVolume = 0;
-    private _inRamp = false;
+    private _running = false;
 
     constructor(player: IMediaPlayer) {
         this._player = player;
+        this._player.volume = this._selectedVolume;
     }
 
     /**
-     * Time, in seconds, that the volume should be lowered.
-     * 
+     * The selected volume.
+     *
      * @remarks
-     * Applications should periodically call `lowerVolume()` to keep the volume lowered. 
+     * Expressed as a value between 0.0 and 1.0. The default value is 1.0.
+     * Can be used for things like volume sliders.
      */
-    public get duration(): number {
-        return this._duration.seconds;
+    public get selectedVolume(): number {
+        return this._selectedVolume;
     }
 
-    public set duration(value: number) {
-        this._duration.seconds = Math.abs(value);
+    public set selectedVolume(value: number) {
+        if (value < 0 || value > 1.0) {
+            throw new Error(`VolumeLimiter: cannot set selectedVolume to ${value}. Level must be between 0.0 and 1.0.`);
+        }
+
+        this._selectedVolume = value;
+        this.startAdjusting();
     }
 
     /**
      * Target level to lower volume to.
-     * 
+     *
      * @remarks
      * Expressed as a value between 0.0 and 1.0. The value is applied based upon the configured
-     * `levelType`. The default value is 0.2.
-     * 
-     * For a level type of `LevelType.fixed` the value is the exact level the volume will be 
-     * lowered to. The default value of 0.2 would cause the volume to be lowered to 0.2.
-     * 
+     * `levelType`. The default value is 0.1.
+     *
+     * For a level type of `LevelType.fixed` the value is the exact level the volume will be
+     * lowered to. The default value of 0.1 would cause the volume to be lowered to 0.1.
+     *
      * For a level type of `LevelType.percentage` the value is the percentage by which the volume
-     * level should be lowered to. The default value of 0.2 would cause the volume to be lowered
-     * to 20% of its starting value.
+     * level should be lowered to. The default value of 0.1 would cause the volume to be lowered
+     * to 10% of its starting value.
      */
     public get level(): number {
```

Comment:
renaming this to `minimumVolume` or something might be better. Whatever it is named, max volume should be named in alignment with it.